packages/store/src/adapter/markdown.ts

```diff
@@ -386,19 +308,31 @@ export class MarkdownAdapter extends BaseAdapter<Markdown> {
           break;
         }
         case 'affine:image': {
-          break;
-        }
-        case 'affine:database': {
+          const blobId = (snapshot.props.sourceId ?? '') as string;
+          await assets?.readFromBlob(blobId);
+          const blob = assets?.getAssets().get(blobId);
+          if (!blob) {
+            break;
+          }
+          const dataURL = await blobToDataURL(blob);
+          context
+            .openNode(
+              {
+                type: 'image',
+                url: dataURL as string,
+                title: null,
+                alt: null,
```

Comment:
Maybe we can try to get alt from `blob.name`.

Reply:
Can I revert this commit? blob.name is function's property rather than blob's property. It returns function's name.
`File` type does have this feature, but unfotunately during the conversion, this property lost.